doc/config.md

```diff
@@ -796,6 +796,12 @@ Default: not set
 
 ### client_tls_ca_file
 
+If the contents of this file are changed without changing the actual
+setting, the new CA file will be used for new connections. Existing
+connections won't be closed though. If it's necessary for security
+reasons that connections using the old CA are recycled ASAP, it's
+advised to run RECONNECT after the reload.
```

Comment:
Let's move this paragraph to the top of the TLS settings section. Also let's change the wording to include RELOAD, because now it sounds like this will happen automatically even without one. Something like this:
```suggestion
If the contents of any of the cert or key files are changed without changing the actual
setting filename in the config, the new file contents will be used for new connections after a RELOAD. Existing
connections won't be closed though. If it's necessary for security
reasons that all connections start using the new files ASAP, it's
advised to run RECONNECT after the RELOAD.
```

Reply:
Probably also good to note that changing any of the settings in the config file will trigger a RECONNECT automatically.

Reply:
[5e27dd2](https://github.com/pgbouncer/pgbouncer/pull/1157/commits/5e27dd239c6f3ef22cf0dd379ab9b3ccdbb65ef4)